src/test/java/org/apache/commons/lang3/builder/DiffBuilderTest.java

```diff
@@ -32,6 +32,31 @@
     
     private static final ToStringStyle SHORT_STYLE = ToStringStyle.SHORT_PREFIX_STYLE;
 
+    private static class IdentifiableTest {
```

Comment:
It's so obvious now that you mention it.  [Patch](https://github.com/j--baker/commons-lang/commit/728ed99c1f21b39f6bae9074112e0ffd31ba7c2b).